googlemock/CMakeLists.txt

```diff
@@ -145,6 +145,14 @@ if (gmock_build_tests)
   # 'make test' or ctest.
   enable_testing()
 
+  if (WIN32)
```

Comment:
This is making an assumption that on Windows the only tool people use is MSVC. Please check other conditions as well ( MinGW)